apps/docs/src/examples/use-gltf-animations/Scene.svelte

```diff
@@ -15,6 +15,7 @@
 
 	const { scene } = useThrelte()
 	scene.background = new Color(0xeae8e2)
+	// TODO: Use `useDraco` instead of `dracoDecoderPath` after #137 is merged
```

Comment:
You can do that immediately, as everything will be deployed at the same time as soon as #137 is merged.